backend/app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' } %>
 
 <% content_for :page_title do %>
-  <i class="fa fa-arrow-right"></i> <%= Spree.t(:customer_details) %>
+  &#x2f; <%= Spree.t(:customer_details) %>
 <% end %>
 
 <% content_for :page_actions do %>
```

Comment:
Shouldn't we remove all empty `content_for :page_actions` blocks? Or do we want to keep them for potential deface overrides?